draft-ietf-gnap-core-protocol.md

```diff
@@ -3213,9 +3213,10 @@ proof (string)
     {{binding-keys}}. The `proof` field is REQUIRED.
 
 jwk (object)
-: Value of the public key as a JSON Web Key {{RFC7517}}. The object MUST
-            contain an "alg" field which is used to validate the signature.
-            The object MUST contain the "kid" field to identify the key.
+: The public key and its properties represented as a JSON Web Key {{RFC7517}}.
+    A JWK MUST contain the "alg" (Algorithm) and "kid" (Key ID) parameters.  The `alg`
+    parameter MUST NOT be `none`.  The "x5c" (X.509 Certificate Chain) parameter MAY
+    be used to provide X.509 representation of the provided public key.
```

Comment:
For consistency's sake, enclose all variables name in `backticks` and literal string values in "quotes".

```suggestion
: The public key and its properties represented as a JSON Web Key {{RFC7517}}.
    A JWK MUST contain the `alg` (Algorithm) and `kid` (Key ID) parameters. The `alg`
    parameter MUST NOT be "none". The `x5c` (X.509 Certificate Chain) parameter MAY
    be used to provide the X.509 representation of the provided public key.
```

Reply:
I would leave two spaces after a period as it was before.

Reply:
A style note from the editors: There should only be one space before period inside of text. Any existing text that uses two spaces before a dot will eventually get cleaned up. It's a minor detail, but if there's a chance to fix or avoid it in a PR, we'll try to do so.